todo_app/data/session_items.py

```diff
@@ -1,43 +1,10 @@
 from flask import session
-import json,requests,uuid,os
-from enum import Enum
-
-class Lists(Enum):
-    TODO = 1
-    FINISHED = 2
-
-class Item:
-    def __init__(self,id = None,status=None,title=None,cardid=None):
-        self.id= id if id is not None else -1
-        self.status= status if status is not None else Lists.TODO
-        self.title= title if title is not None else 'List saved todo items'
-        self.cardid= cardid
-    def unstarted(self):
-        return self.status == Lists.TODO.value
-    def description(self):
-        return Lists(self.status).name  
-    def __str__(self):
-     return "id:" + str(self.id)  
-
-def init_board():
-    boardName = uuid.uuid4().hex
```

Comment:
Yes Jack, i thought that the idea was that each visitor would have his own board, never crossed my mind that it would  be shared with others. There is a limit of 5 boards, so I guess a paid subscription should be in place for this.

Reply:
I have amended this now, as probably my approach that allows each user to have a different board can cause future problems if i deviate from the expected behaviour. Now one board is created / used for all users.